Source/Authorization/OAuthService.swift

```diff
@@ -72,10 +68,10 @@ class OAuthService {
     
     func renewAccessToken(oauthToken: OAuthToken, completion: @escaping (Result<OAuthToken, Error>) -> Void) {
         guard let jwt = JWTUtility.refreshTokensRequestJWT(refreshToken: oauthToken.refreshToken.value, configuration: configuration) else {
-            fatalError("Invalid pre-authorization request JWT")
+            fatalError("Invalid refresh tokens request JWT")
```

Comment:
We should probably steer away from fatal errors in the context of the SDK, because this will cause the host application to exit.